<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsPage</class>
 <widget class="QWidget" name="ToolsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolsPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1">
     <item>
      <widget class="QTextBrowser" name="tbShow"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="cbName"/>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="serialPage">
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>波特率</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbBaudRate">
                   <item>
                    <property name="text">
                     <string>9600</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>19200</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>38400</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>57600</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>115200</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="minimumSize">
                        <size>
                         <width>50</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>50</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>数据位</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cbDataBit">
                       <property name="currentIndex">
                        <number>3</number>
                       </property>
                       <item>
                        <property name="text">
                         <string>5</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>6</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>7</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>8</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="minimumSize">
                        <size>
                         <width>50</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>50</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>停止位</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cbStopBit">
                       <item>
                        <property name="text">
                         <string>1</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>2</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>1.5</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnOpenSerial">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>打开串口</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="netPage">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>远程</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lePeerIp"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lePeerPort">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>本地</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="leLocalIp"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="leLocalPort">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QPushButton" name="btnNetStart">
                   <property name="text">
                    <string>侦听</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnNetStop">
                   <property name="text">
                    <string>断开</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>广告</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSend">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="cbHexShow">
             <property name="text">
              <string>Hex显示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbHexSend">
             <property name="text">
              <string>Hex发送</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbAddReturn">
             <property name="text">
              <string>加回车换行</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClienShow">
             <property name="text">
              <string>清空显示</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="cbAddTime">
             <property name="text">
              <string>加时间戳和分包显示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbSendAtTime">
             <property name="text">
              <string>定时发送</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leTime">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>ms/次</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="teSendData"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
